.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-20.04
+          - ubuntu-18.04
```

Comment:
Hopefully this is a temporary workaround and we can get back on to 20.04 once this is fixed.